devine/commands/dl.py

```diff
@@ -918,6 +918,10 @@ def cleanup():
                             progress(downloaded="Decrypting", completed=0, total=100)
                             drm.decrypt(save_path)
                             track.drm = None
+                            if isinstance(track, Subtitle) and track.auto_fix_encoding:
+                                data = track.path.read_bytes()
+                                content = Subtitle.fix_encoding(data)
+                                track.path.write_bytes(content)
```

Comment:
Is there a reason this is within the `if drm:` branch of code? This would mean the subtitle would have to be DRM-protected, which it wouldn't have been.

Reply:
I've moved it outside if the if drm tree, and after it. Ultimately same result but now runs even if the Subtitle track did not have DRM (the most likely scenario).

Reply:
Ah, you are correct, that was an oversight on my part